demo/project_demo04-SecOps/run.py

```diff
@@ -14,8 +14,10 @@
 SCRIPT_DIR = os.path.join(os.path.dirname(__file__))
 
 
-def prepare():
-    cmd = ["cargo", "run", "--release"]
+def prepare(args=[2000000]):
+    assert len(args) == 1, "Expected one '--prepare-args' argument for num_pipelines"
```

Comment:
This will break `demo/demo.sh`, which calls this script without arguments.  Is it possible to either fixup that script or allow this to run without `--prepare-args`?

Reply:
It works without prepare-args too. Note the default value for the parameter.

Reply:
got it